Sources/Phakchi/ServiceClient.swift

```diff
@@ -4,140 +4,133 @@ private let adminHeaders = [
     "X-Pact-Mock-Service": "true",
     "Content-Type": "application/json",
 ]
-typealias CompletionHandler = (NSData?, NSHTTPURLResponse?, NSError?) -> Void
+typealias CompletionHandler = (Data?, HTTPURLResponse?, Error?) -> Void
 typealias VerificationCompletionHandler = (Bool) -> Void
 
 protocol BaseServiceClient {
-    var baseURL: NSURL { get }
+    var baseURL: URL { get }
 }
 
 extension BaseServiceClient {
     typealias buildRequestBlock = (NSMutableURLRequest) -> Void
-    private func buildPactRequest(to endpoint: String,
-                                     method: HTTPMethod,
-                                     headers: [String: String] = adminHeaders,
-                                     block: buildRequestBlock? = nil) -> NSMutableURLRequest {
-        #if swift(>=2.3)
-            guard let endpointURL = baseURL.URLByAppendingPathComponent(endpoint) else {
-            fatalError("Invalid endpoint \(endpoint)")
-            }
-        #else
-            let endpointURL = baseURL.URLByAppendingPathComponent(endpoint)
-        #endif
-        let request = NSMutableURLRequest(URL: endpointURL)
-        request.HTTPMethod = method.rawValue
+    fileprivate func buildPactRequest(to endpoint: String,
```

Comment:
I tried to change them to `fileprivate`. However it doesn't work correctly.
Because sub classes(`ControlServiceClient`, `MockServiceClient`) use this method.